controllers/api.controller.js

```diff
@@ -0,0 +1,16 @@
+const { selectSnacks } = require("../model/model");
+
+function invalidPathHandler(req, res, next) {
+  res.status(200).send({ message: "Path Does Not Exist" });
```

Comment:
lets have our error handlers in a separate error handling file ( also this is sending the wrong code back)